inst/substitutions/hmdb_subst_negative.txt

```diff
@@ -0,0 +1,915 @@
+"name"	"md"	"minmass"	"maxmass"	"leftend"	"rightend"	"LBint"	"LBslope"	"UBint"	"UBslope"
```

Comment:
maybe better if we remove the columns minmass and maxmass from all files - we don't use them at present, and the smaller the table the faster the subsetting operations.